models/database.go

```diff
@@ -119,6 +119,11 @@ var databaseSchema = [][]string{
 			CONSTRAINT runs_on_node_id_only_for_pmm_agent CHECK ((runs_on_node_id IS NULL) <> (agent_type='` + string(PMMAgentType) + `'))
 		)`,
 
+		`ALTER TABLE agents 
```

Comment:
it's still a part of another migration. 
please take a look at this line, it creates new migration
https://github.com/percona/pmm-managed/pull/254/files#diff-087672b3a344779900ce93f4af23902dR161